### DISCUSSION TITLE
Miss# challenge mod similiar to acc challenge

### DISCUSSION THREAD

**Discussion by Stoneybeans** [ppy/osu#27198] [Ideas]
Basically it'd be really nice to have a mod where I can only miss X number of objects before failing. Specifically on aim oriented maps where the rhythm is relatively basic and the challenge comes moreso from aim then anything else and i'm trying to push skillcap on aim. A flat amount makes sense here since that's just how this game is played, 1x miss plays matter regardless of the length of map, though I see no reason you couldn't do either a % or flat amount.

 Sliderends could also be an interesting toggle on this for slider aim so that if i miss enough slider ends they also end my run. This makes plays on slider heavy/gimmick maps a lot more tight as well. Obviously i'm ignoring the coding side of things, hopefully counting sliderends as misses only in cases of this mods usage wouldn't be too difficult.

 From my perspective this would make practicing aim oriented maps waaaaaaaaaaay more interesting since i can simply say on NM1~ tourny maps only allow for X misses instead of a flat % acc amount and actually feel like i'm improving instead of just getting a bunch of misses and 98% acc thus getting a false impression on improvement on what those maps are testing.

**Comment by OwenCMYK** [comment]
I think this would be best as a slider in the mod configuration for the "Sudden Death" mod. But now that I think about it, it makes me wonder why Accuracy Challenge and Perfect aren't combined into one mod, considering Perfect is just the same as 100% accuracy challenge, though that's somewhat off-topic.

**Reply by Stoneybeans** [reply]
The only issues i see is with the nomenclature of "Sudden Death" and "Perfect" conflicting with lowering the requirements for each mod. Sudden death indicates if you miss you die, but what i'm proposing is something with more leeway based on your settings. Perfect makes little sense unless it requires "Perfection" tbh.

**Reply by OwenCMYK** [reply]
Yeah, but given that "Double Time" and "Half Time" are two other mods with adjustment, I think it might be fitting. Then again, I don't code on this project, so I guess we'll see what the devs think
